### PR TITLE
doing: init at 1.0.10pre

### PR DESCRIPTION
**Pull request by ktf**
###### Motivation for this change

Adds the `doing` application.
 
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/doing/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{ lib, bundlerEnv, ruby
+}:
+
+# Bundix:
+# nix-shell -p bundix zlib
+bundlerEnv rec {
+  name = pname + "-" + version;
```
Comment (ryantm) [inline pkgs/applications/misc/doing/default.nix]: This line can be removed because `bundlerEnv` will handle it.

**Review by ryantm — CHANGES_REQUESTED** [review]

**Comment by ryantm** [comment]
It looks like this is missing an addition to `all-packages.nix`, or how do you build this application?

**Comment by ktf** [comment]
right... I've it in my own overlay... Will fix it...

**Review by xeji — APPROVED** [review]
Feedback was addressed. LGTM.

**Pull request merged**